server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -265,21 +265,27 @@ async fn create_message(
             ..Default::default()
         };
         msg_dests.push(msg_dest);
-        tasks.push(
-            MessageTask::new_task(
-                msg.id.clone(),
-                app.id.clone(),
-                endp.id, MessageAttemptTriggerType::Scheduled));
+        endpoint_ids.push(endp.id.clone());
     }
     if !msg_dests.is_empty() {
         messagedestination::Entity::insert_many(msg_dests)
             .exec(db)
             .await?;
```

Comment:
Don't save them here, only save them in the worker (to reduce latency here). You can just calculate them again easily there.

Reply:
Do we still want messagedestination rows? If we're going to just calculate this on the fly, I'm not sure what their use is; messageattempt acts as our record of what actually occurred.

But, if we're going to still want these rows, I feel like this is where that write belongs? Otherwise we are doing an upsert or conditional insert in the worker to only write these rows the first time a task is processed, which I think is awkward - the latency isn't _that bad_ is it?

Reply:
We will eventually get rid of messagedestination because they aren't really needed. Though they are needed for now unfortunately (need to make a few more changes before we can get rid of them).

I think these can be written in the worker before you call each `dispatch`.